modules/reference/pages/properties/broker-properties.adoc

```diff
@@ -137,6 +137,26 @@ TIP: For backward compatibility, `true` is the default. Redpanda recommends usin
 
 ---
 
+=== fips_mode
+
+Controls whether Redpanda starts in FIPS mode.  This property allows for three values: 
+
+* Disabled - Redpanda does not start in FIPS mode.
+
+* Permissive - Redpanda performs the same check as enabled, but a warning is logged, and Redpanda continues to run. Redpanda loads the OpenSSL FIPS provider into the OpenSSL library. After this completes, Redpanda is operating in FIPS mode, which means that the TLS cipher suites available to users are limited to the TLSv1.2 and TLSv1.3 NIST-approved cryptographic methods.
```

Comment:
These options are also listed in https://docs.redpanda.com/beta/manage/security/fips-compliance/#configure-fips
One important piece of guidance is missing here:
permissive: This setting is a safety value option only, and should not be used in a production environment. If specified, Redpanda logs a WARNING, but continues operations even if the underlying operating system is not configured for FIPS. If set, your Redpanda instance is not running in FIPS compliance mode.

Reply:
@michael-redpanda Can you please weigh in on these descriptions? I'm thinking they should be identical to what we have in https://docs.redpanda.com/beta/manage/security/fips-compliance/#configure-fips. AFAIK, the note in permissive is kind of critical, no?

Reply:
@Feediver1 this section is already out and should not be reviewed with this PR. Rebasing to fix the issue..

Reply:
I'll file a new PR to include your warning